init.d.optee

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# /etc/init.d/optee
+#
+# Start/stop tee-supplicant (OP-TEE normal world daemon)
+#
+case "$1" in
+    start)
+	if [ -e /bin/tee-supplicant -a -e /dev/teepriv0 ]; then
```

Comment:
99.9% of the times when we're using tee-supplicant in our setup's we simply use `tee-supplicant &`. So, at this moment I don't think it's worth taking care of extra args etc, even though Igor is right in principle.